hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancer.java

```diff
@@ -519,6 +517,31 @@ public void testRegionLoadCost() {
     assertEquals(2.5, result, 0.01);
   }
 
+  @Test
+  public void testRegionLoadCostWhenDecrease() {
```

Comment:
Yeah, good idea, @bbeaudreault. It's reasonable to test the [1, 2, 1, 4], since it's rare to see 0 in the real scenarios, and I used 0 to describe the issue more clearly. Thanks.